p2p/nodeconfig/config.go

```diff
@@ -56,25 +56,25 @@ func (d *duration) Duration() (duration time.Duration) {
 	return dur
 }
```

Comment:
We added this duration workaround because `urfave/cli` had a bug with loading go's `Duration` type. if cobra is capable of unmarshalling into go's `Duration` we should remove this.